### DISCUSSION TITLE
Run pcm in a docker container with cpuset

### DISCUSSION THREAD

**Discussion by scse-l** [opcm/pcm#581] [Q&A] [answered]
I run pcm-memory in a docker container. It throws exception: 
> ERROR: pthread_setaffinity_np for core 0 failed with code 22.
I guess the reason is that the container uses only a few cores. pcm-memory tries to get data from all cpu cores.
How can I run pcm-memory in a docker container?

**Comment by opcm** [comment, marked as answer]
it is possible to run pcm in a docker container but w/o the core restriction. Here are some hints: https://github.com/intel/pcm/blob/master/doc/DOCKER_README.md
